### DISCUSSION TITLE
Possibility to edit thankyou template from the WordPress site editor

### DISCUSSION THREAD

**Discussion by j-falk** [woocommerce/woocommerce#44975] [WooCommerce, Blocks, and Site Editing]
At Krokedil we build a plugin called [Post Purchase Upsell for WooCommerce](https://krokedil.com/product/post-purchase-upsell-for-woocommerce/) (currently in beta) that offer a customer a possibility to add products to their previously created order from the thank you page.

We now are in some discussions about how to enable customization and integration with other plugins and I then came to think about what plans there are for block support for this template.

Do you have any idea of when we could see support for editing the [thankyou template](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/templates/checkout/thankyou.php) from WooCommerce in the WordPress site editor?

**Comment by nerrad** [comment]
This is something the team is currently planning to work on in 2023 (cc @ralucaStan).

**Reply by pierorocca** [reply]
Hi @j-falk ,

In 2023 we'll be focused on end-to-end site editing Block support and design work has started on the thank you page. Would love to get your feedback once the timing allows if you're interested?

**Reply by j-falk** [reply]
Hi @pierorocca, sounds great and I would love to give feedback and test when there is something to look at 🙌
